src/app/api/vendors/[id]/check-in/route.ts

```
@@ -0,0 +1,85 @@
+            .from('vendor_milestones')
+            .update({
+                actual_arrival_timestamp: now.toISOString(),
+                status: 'completed',
+            })
+            .eq('id', milestoneId);
+
+        if (updateError) {
+            throw new Error(updateError.message);
+        }
+
+        // 5. Execute Stripe Connect transfer with slashed amount (mocked)
```

ESLint

[error] Delete `····`

(prettier/prettier)